### DISCUSSION TITLE
No roundabout specific announcements

### DISCUSSION THREAD

**Discussion by ReinerMeyer** [osmandapp/OsmAnd#19012] [Maps Related]
Dear all,
recently I used a track for navigation (car profile). To create this route I used Brouter Web Client. This tool uses exactly the ways stored in OSM database.
I noticed that there are no roundabout-specific announcements (exit number).
 
However, I get the roundabout-specific instructions when I enter a point (e.g. Favorite POI) as destination for navigation.

Either I am not using the correct procedure or it is not possible with OSMAND.
What can I do to solve this problem?

Would it be possible to save several points together as one favorite POI so that the navigation algorithm recalculates the route each time similar like using only one favorite POI?
I know from Garmin that you can create and save a route by entering different points. No track is created that you can call up.
This method is described as "GPS Exchange Format" in https://en.wikipedia.org/wiki/GPS_Exchange_Format

Regards
Reiner

**Comment by DmitryAlexei** [comment]
@ReinerMeyer Please try *Attach to roads* option. For more information see https://osmand.net/docs/user/navigation/setup/gpx-navigation#attach-to-roads
